enterprise/frontend/src/embedding-sdk/cli/utils/ask-for-db-connection-info.ts

```diff
@@ -37,8 +37,10 @@ export async function askForDatabaseConnectionInfo(options: Options) {
 
     const shouldShowField =
       !visibleIf ||
+      // some of the entries might be undefined, not false, so we need to compare
```

Comment:
yep, it was just to get things working. I won't push this

Reply:
I'll resolve this comment since you pulled this out and it will be fixed in [fix(sdk): Fix CLI not showing Postgres password field](https://github.com/metabase/metabase/pull/46876#top) anyway.